createdisk.sh

```diff
@@ -181,26 +154,20 @@ libvirtDestDir="${destDirPrefix}_libvirt_${destDirSuffix}"
 rm -fr ${libvirtDestDir} ${libvirtDestDir}.crcbundle
 mkdir "$libvirtDestDir"
 
-if [ $BUNDLE_TYPE != "microshift" ]; then
-    create_qemu_image "$libvirtDestDir" "${VM_PREFIX}-base" "${VM_NAME}"
-    mv "${libvirtDestDir}/${VM_NAME}" "${libvirtDestDir}/${SNC_PRODUCT_NAME}.qcow2"
-else
-    sparsify_lvm "${libvirtDestDir}"
-fi
+function copy_qemu_image {
+    if [ $BUNDLE_TYPE != "microshift" ]; then
+        create_qemu_image "$libvirtDestDir" "${VM_PREFIX}-base" "${VM_NAME}"
+        mv "${libvirtDestDir}/${VM_NAME}" "${libvirtDestDir}/${SNC_PRODUCT_NAME}.qcow2"
+    else
+        sparsify_lvm "${libvirtDestDir}"
+    fi
+}
+copy_qemu_image
```

Comment:
There's already a `create_qemu_image` which copies and sparsifies, please don't add an additional `copy_qemu_image` which calls `create_qemu_image` and also sparsifies in other cases. Let's use this opportunity to group all this code :)

Reply:
@cfergeau you mean `create_qemu_image` should also work with LVM image type?